samples/keys/identity_key_generation/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Building TFM requires Cmake 3.15.
```

Comment:
The line below enforces that CMake is at least 3.20.0, so I don't see what relevance the TF-M requirement has.

Drop this line

Reply:
Also, I don't see any _ns boards in the integration platforms list.

Is it even supported to use this sample wtih TF-M ?

Reply:
This sample doesn't use TF-M and expects a secure-only build
The comment probably came from some copy-paste and has been removed